src/common/repositories/auth/useAuthRepository.tsx

```diff
@@ -0,0 +1,18 @@
+import { Ssignup } from '../../services/auth/authService';
+import { TSignupRequest } from '../../services/auth/types/requests/TSignupRequest';
+import { TSignupResponse } from '../../services/auth/types/responses/TSignupResponse';
+import { TuseAuthRepository } from './types/TuseAuthRepository';
+
+// Define as a normal function, not a React.FC
+const useAuthRepository = (props: TuseAuthRepository) => {
```

Comment:
```suggestion
const useAuthRepository = () => {
```
Si c'est destiné à être vide, autant ne pas le mettre